site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -174,7 +174,7 @@
             <label class="inline-block" for="Announcement">{{ announcement_text }}</label>
             <input type="checkbox" class="thread-announcement-checkbox" name="Announcement" value="Announcement" data-ays-ignore="true"/>
             <label class="inline-block" for="Announcement">{{ "Pin Thread" }}</label>
-            <input type="checkbox" class="thread-announcement-checkbox" name="pinThread" value="pinThread" data-ays-ignore="true"/>
+            <input type="checkbox" class="pinThread-checkbox" name="pinThread" value="pinThread" data-ays-ignore="true"/>
```

Comment:
It looks like the old class `thread-announcement-checkbox` had some css attached to it, see forum.css line 411. You might want to switch that line of css to also include `pinThread-checkbox` so that this change does not mess with the css